README.md

```diff
@@ -67,10 +67,77 @@ $ brew upgrade dep
 ## Unit testing
 
 1. Clone the `codewind-installer` repository.
-2. Use the `cd` command to go to the test directory. The `utils.go` tests are located in the `utils/utils_test.go` file.
+2. Use the `cd` command to go to a directory with test files in. For example, the `utils.go` tests are located in the `utils/utils_test.go` file.
 3. To run the tests, enter the `go test -v` command in the command line window and wait for the tests to finish.
 4. For any other unit tests, the same steps apply, but the directory might change.
 
+## Bats-core testing
+
+1. Set up your environment by installing bats-core as per the bats-core instructions found at <https://github.com/bats-core/bats-core>.
+2. Clone the `codewind-installer` repository.
+3. Use the `cd` command to go to the top level project directory.
+4. Ensure your system environment is clean by having no Codewind images installed or containers running.
+5. To run the tests, enter the `bats integration.bats` command in the command line window and wait for the tests to finish.
+
+## CLI Commands
+
+|Command         |Alias         |Usage                                                           |
+|----------------|--------------|----------------------------------------------------------------|
+|project         |`-`           |'Manage Codewind projects'                                      |
```

Comment:
Not clear on how to use the dash alias here or in the lines below.  Are you saying that I can type a dash and that means project or means start or means stop (which seems confusing) or does it mean "Not applicable" in which case it may be better to just keep that cell in the table empty.